assets/src/scripts/web-services/models.js

```diff
@@ -66,13 +68,11 @@ export const getTimeSeries = function({sites, params=null, startDate=null, endDa
 };
 
 export const getPreviousYearTimeSeries = function({site, startTime, endTime, parameterCode}) {
+    const hoursInOneYear = 8760;
     parameterCode = parameterCode ? [parameterCode] : null;
+    const lastYearStartTime = new DateTime.fromMillis(startTime).minus({hours: hoursInOneYear});
```

Comment:
Something I've discovered while doing the jest test conversion: we shouldn't be using new DateTime.fromMillis. The DateTime.fromMillis is a static method on the DateTime class therefore doesn't need the new and in fact this causes an error when running jest tests. So please remove the "new" here and in line 74

Reply:
Ah, very interesting and good to know.